.gitmodules

```diff
@@ -1,16 +1,16 @@
 [submodule "vendor/postgres-v14"]
 	path = vendor/postgres-v14
 	url = https://github.com/neondatabase/postgres.git
-	branch = REL_14_STABLE_neon
+	branch = 28934-pg-dump-schema-no-create-v14
```

Comment:
should be reverted?

Reply:
We should merge the changes into `neondatabase/postgres` first. I don't see a pr though.

Reply:
Step 16 says to set the branch back to the stable one or am I missing something?
```
git submodule set-branch --branch REL_15_STABLE_neon vendor/postgres-v15
```